sdk/src/test/scala/io/horizen/history/validation/ConsensusValidatorOmmersTest.scala

```diff
@@ -488,6 +649,129 @@ class ConsensusValidatorOmmersTest
 
   }
 
+  @Test
+  def switchingEpochOmmersWithSubOmmersValidationAfterConsensusParamsFork(): Unit = {
+    // Mock Consensus epoch info data
+    val postForkEpochNonceBytes: Array[Byte] = new Array[Byte](8)
+    scala.util.Random.nextBytes(postForkEpochNonceBytes)
+    val postForkFullConsensusEpochInfo = FullConsensusEpochInfo(mock[StakeConsensusEpochInfo],
+      NonceConsensusEpochInfo(byteArrayToConsensusNonce(postForkEpochNonceBytes)))
+
+    val preForkEpochNonceBytes: Array[Byte] = new Array[Byte](8)
+    scala.util.Random.nextBytes(preForkEpochNonceBytes)
+    val preForkFullConsensusEpochInfo = FullConsensusEpochInfo(mock[StakeConsensusEpochInfo],
+      NonceConsensusEpochInfo(byteArrayToConsensusNonce(preForkEpochNonceBytes)))
+
+    val switchedOmmersCurrentEpochNonceBytes: Array[Byte] = new Array[Byte](8)
+    scala.util.Random.nextBytes(switchedOmmersCurrentEpochNonceBytes)
+    val switchedOmmersFullConsensusEpochInfo = FullConsensusEpochInfo(postForkFullConsensusEpochInfo.stakeConsensusEpochInfo,
+      NonceConsensusEpochInfo(byteArrayToConsensusNonce(switchedOmmersCurrentEpochNonceBytes)))
+
+
+    /* Test 1: Valid Ommers with subommers in correct order from the previous and current epoch as VerifiedBlock
+       Notation <epoch_number>/<slot_number>
```

Comment:
Consider changing the slot numbers postFork to numbers greater than the previous epoch length (e.g. 15 instead of 5) just as an additional check.